dapp/src/components/WalletSelectContent.js

```diff
@@ -89,23 +96,30 @@ const WalletSelectContent = ({}) => {
             'Connect a wallet to get started'
           )}
         </h2>
-        {['MetaMask', 'Ledger', 'MyEtherWallet', 'WalletConnect'].map(
-          (name) => {
-            return (
-              <button
-                key={name}
-                className="connector-button d-flex align-items-center"
-                onClick={() => onConnect(name)}
-              >
-                <div className="col-2">
-                  <img src={`/images/${name.toLowerCase()}-icon.svg`} />
-                </div>
-                <div className="col-8">{name}</div>
-                <div className="col-2"></div>
-              </button>
-            )
-          }
-        )}
+        {[
+          'MetaMask',
+          'Ledger',
+          'CoinbaseWallet',
+          'MyEtherWallet',
+          'WalletConnect',
```

Comment:
nit: can you change the order of wallets as requested by Micah in the original ticket please: 
`MetaMask, Ledger, Coinbase Wallet, WalletConnect, MEW wallet`

Reply:
Hey, just made those changes! Hope everything is fine now!

Reply:
Perfect thanks a lot.